docs/guides/installation.zh_CN.md

```diff
@@ -92,9 +92,9 @@ Rocky-8.4-x86_64-minimal.iso: OK
 _安装信息摘要_  是一个集合多种设置的区域，您可以在其中做出有关要安装的操作系统的重要决定。
 
 界面大致分为以下几部分：
-    • _本地化_（键盘、语言支持以及时间和日期）
-    • _软件_（安装源和软件选择）
-    • _系统_（安装目的地以及网络和主机名）
+- _本地化_（键盘、语言支持以及时间和日期）
```

Comment:
@jimcat8 dashes ‘-‘ are accepted Markdown code for unordered lists, for the Rocky style guide I believe ‘*’ asterisks are preferred. I don’t think @sspencerwire will reject the PR for this point, but obviously he will have the final say.

Reply:
@justasojourner is correct: '*' are the preferred method, but I'll merge these changes anyway. If you notice any problems, please submit a new PR to fix the fix. Thanks again for your contributions!